.github/workflows/development.yaml

```diff
@@ -3,145 +3,166 @@ on:
+  #         echo "PKG_NAME=${PKG_NAME}" >> $GITHUB_ENV
+  #         echo "PKG_VERSION=${PKG_VERSION}" >> $GITHUB_ENV
+  #     - name: Add image artifact
+  #       uses: actions/upload-artifact@v2
+  #       with:
+  #         name: image-${{env.PKG_NAME}}-${{env.PKG_VERSION}}-py${{matrix.py_ver}}-${{matrix.distro}}
+  #         path:
+  #           "image-${{env.PKG_NAME}}-${{env.PKG_VERSION}}-py${{matrix.py_ver}}-\
+  #           ${{matrix.distro}}.tar.gz"
+  #         retention-days: 1
+  #     - if: matrix.py_ver == '3.8' && matrix.distro == 'alpine'
+  #       name: Add pip artifacts
+  #       uses: actions/upload-artifact@v2
+  #       with:
+  #         name: pip-${{env.PKG_NAME}}-${{env.PKG_VERSION}}
+  #         path: dist
+  #         retention-days: 1
+  # publish-release:
+  #   if: github.event_name == 'push'
+  #   needs: build
+  #   runs-on: ubuntu-latest
+  #   env:
+  #     TWINE_USERNAME: ${{secrets.twine_username}}
+  #     TWINE_PASSWORD: ${{secrets.twine_password}}
+  #   outputs:
+  #     release_upload_url: ${{steps.create_gh_release.outputs.upload_url}}
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - name: Determine package version
+  #       run: |
+  #         PKG_NAME=$(python3 -c "print([p for p in __import__('setuptools').find_packages() if '.' not in p][0])")
+  #         SDIST_PKG_NAME=$(echo ${PKG_NAME} | sed 's|_|-|g')
+  #         PKG_VERSION=$(cat ${PKG_NAME}/version.py | awk -F\' '/__version__ = / {print $2}')
+  #         echo "PKG_NAME=${PKG_NAME}" >> $GITHUB_ENV
+  #         echo "PKG_VERSION=${PKG_VERSION}" >> $GITHUB_ENV
+  #         echo "SDIST_PKG_NAME=${SDIST_PKG_NAME}" >> $GITHUB_ENV
+  #     - name: Get changelog entry
+  #       id: changelog_reader
+  #       uses: guzman-raphael/changelog-reader-action@v5
+  #       with:
+  #         path: ./CHANGELOG.md
+  #         version: ${{env.PKG_VERSION}}
+  #     - name: Create GH release
+  #       id: create_gh_release
+  #       uses: actions/create-release@v1
+  #       env:
+  #         GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+  #       with:
+  #         tag_name: ${{steps.changelog_reader.outputs.version}}
+  #         release_name: Release ${{steps.changelog_reader.outputs.version}}
+  #         body: ${{steps.changelog_reader.outputs.changes}}
+  #         prerelease: ${{steps.changelog_reader.outputs.status == 'prereleased'}}
+  #         draft: ${{steps.changelog_reader.outputs.status == 'unreleased'}}
+  #     - name: Fetch image artifact
+  #       uses: actions/download-artifact@v2
+  #       with:
+  #         name: image-${{env.PKG_NAME}}-${{env.PKG_VERSION}}-py3.8-alpine
+  #     - name: Fetch pip artifacts
+  #       uses: actions/download-artifact@v2
+  #       with:
+  #         name: pip-${{env.PKG_NAME}}-${{env.PKG_VERSION}}
+  #         path: dist
+  #     - name: Publish pip release
+  #       run: |
+  #         export HOST_UID=$(id -u)
```

Comment:
Do not comment out code. If it's not necessary, then remove it.

Reply:
It is necessary for publishing. I'm just disabling the workflows related to publishing temporarily so I can deploy to github pages on my fork.